FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Angela Lin on 9/6/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedImageMapper {
+	private struct NetworkFeedImage: Decodable {
+		let id: UUID
+		let description: String?
+		let location: String?
+		let url: URL
+
+		private enum CodingKeys: String, CodingKey {
+			case id = "image_id"
+			case description = "image_desc"
+			case location = "image_loc"
+			case url = "image_url"
+		}
+	}
+
+	private struct NetworkFeedImageContainer: Decodable {
```

Comment:
Suggestion:
Since this struct is private, there's no need to give extra context in the name of the type. It can be simplified to just `Container`
```suggestion
	private struct Container: Decodable {
```